### DISCUSSION TITLE
8.6.1 (Dimming and power fixes)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#427] [Announcements]
###  Changes into this release:

`All tools`:
- Bug fixed: non-terminated string can be loaded from registry, providing incorrect string value (ghost chars at the end).

`All fan-control GUI tools`:
- Bug fixed: Incorrect power mode can be selected in some cases

`AlienFX-GUI`:
- MS fix screen dimming support (was not work for years). Now, GUI support screen dimming (lights will be dimmed as well) in case "Follow screen state" enabled into Options tab.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/8.6.1'>8.6.1 (Dimming and power fixes)</a>.</em>

**Comment by roboboredom** [comment]
Thanks for all your hard work! This app is an absolute life saver for my alienware m16 and I use it every day. 🥇

**Comment by leoinpython** [comment]
I'm using Area 51M Gen1, I find that there's no light control for the back of the Laptop
![vpavic_190102_3148_0136](https://github.com/T-Troll/alienfx-tools/assets/43802162/0ead0aad-90b7-4fbc-bb8e-fa83069a4ea7)
no matter I use all lights off mode, it doesnt turn off that backlight, which is quite annoying as its pretty bright in the dark, this light can be controlled by AWCC but unfortunately my AWCC is completely useless now so I've been trying to use alienfx for fan control and light control, will there be a fix for this?

**Reply by T-Troll** [reply]
You should have 2 light devices - seems like one is not set correctly and not controlled. Check "Devices and grids" tab.

**Comment by Schurke** [comment]
Fantastic work. Just rolled this out on a 2023 M18 R1 (just colours). Mapping the keyboard was tedious but being able to set the lights without AWCC is totally worth it.

**Reply by T-Troll** [reply]
Thank you! 
You can help to all m18 owners - please share light mappings to me, i'll add it into application database.
To do so, just press "Save lights and grids" button at Devices tab and share .csv file it generates.

**Reply by Schurke** [reply]
Will do when it's a bit more solid - I'm still missing some states on the power button: I was able to get plugged-in vs running on battery locked down but haven't found the charging state (or the critical state) yet.

**Comment by leoinpython** [comment]
Just did, please see if this is the correct file, thanks.
